homeassistant/components/notify/telegram.py

```diff
@@ -102,6 +103,8 @@ def send_message(self, message="", **kwargs):
             return
         elif data is not None and ATTR_LOCATION in data:
             return self.send_location(data.get(ATTR_LOCATION))
+        elif data is not None and ATTR_DOCUMENT in data:
+            return self.send_document(data.get(ATTR_DOCUMENT, None)[0])
```

Comment:
There is no need to specify a default value for `get(…)` because a) default is already `None` and b) your if-statement validates that the key exists.

However, you are making the assumption that ATTR_DOCUMENT is a list and I am not sure why? Wouldn't it make more sense to just pass the value of ATTR_DOCUMENT in ?